commands/update_party_profile.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2023 Gobalsky Labs Limited
```

Comment:
2024?

Reply:
We have to update the linter then. Another PR I believe.